Keyboardio/Model01/experimental/Model01/Model01.ino

```diff
@@ -452,27 +447,32 @@ KALEIDOSCOPE_INIT_PLUGINS(
 
   // The chase effect follows the adventure of a blue pixel which chases a red pixel across
   // your keyboard. Spoiler: the blue pixel never catches the red pixel
-  LEDChaseEffect,
+  // TODO(anyone): Currently disabled to save space
+  // LEDChaseEffect,
 
   // These static effects turn your keyboard's LEDs a variety of colors
-  //solidRed, solidOrange, solidYellow, solidGreen, solidBlue, solidIndigo, solidViolet,
+  // TODO(anyone): Currently disabled to save space
+  // solidRed, solidOrange, solidYellow, solidGreen, solidBlue, solidIndigo, solidViolet,
 
   // The breathe effect slowly pulses all of the LEDs on your keyboard
-  LEDBreatheEffect,
+  // TODO(anyone): Currently disabled to save space
+  // LEDBreatheEffect,
 
   // The AlphaSquare effect prints each character you type, using your
   // keyboard's LEDs as a display
   // TODO(anyone): Currently disabled to save space
   // AlphaSquareEffect,
 
   // The stalker effect lights up the keys you've pressed recently
-  StalkerEffect,
+  // TODO(anyone): Currently disabled to save space
+  // StalkerEffect,
 
   // The Colormap effect makes it possible to set up per-layer colormaps
   ColormapEffect,
 
   // The numpad plugin is responsible for lighting up the 'numpad' mode
   // with a custom LED effect
+  // TODO(anyone): Currently disabled to save space
```

Comment:
Minor nitpick: The `TODO` doesn't match the code, `NumPad` isn't disabled.

Reply:
Oops! I was turning stuff on and off experimentally to see what impact it had, and thought it would make sense to cut NumPad, but then changed my mind. _I_ wouldn't use it here, because the numpad layer isn't configurable by Chrysalis (I think?), but it has more utility than some of the other LED plugins, so I put it back in.